test/e2e/kcm_controller.go

```
@@ -0,0 +1,268 @@
+	output, err = oc.WithoutNamespace().Run("describe").Args("quota", "quota43039", "-n", ns).Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	o.Expect(output).To(o.ContainSubstring("openshift.io/imagestreams  0     10"))
+}
+
+func testCrossNamespaceOwnerRefDeleted() {
+	oc := testlib.NewOC()
+	deployT := filepath.Join(testlib.FixturePath("workloads"), "deploy_duplicatepodsrs.yaml")
+
+	err := oc.WithoutNamespace().Run("new-project").Args("p43092-1").Execute()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	defer oc.WithoutNamespace().Run("delete").Args("project", "p43092-1").Execute()
```

golangci-lint (2.12.2)

[error] Error return value of `(*github.com/openshift/cluster-kube-controller-manager-operator/test/library.ocCommand).Execute` is not checked

(errcheck)